inc/saihostif.h

```diff
@@ -322,6 +322,16 @@ typedef enum _sai_hostif_trap_type_t
     /** Default packet action is forward */
     SAI_HOSTIF_TRAP_TYPE_IPV6_NEIGHBOR_ADVERTISEMENT = 0x00002013,
 
+    /**
+     * @brief Intermediate System-to-Intermediate System (IS-IS) protocol
+     * traffic:
+     * L1 IS: 01:80:c2:00:00:14
+     * L2 IS: 01:80:c2:00:00:15
+     * All IS: 09:00:2b:00:00:05
```

Comment:
Are those mac addresses part of this protocol standard ? also please make this more like this:

```
 * @brief Intermediate System-to-Intermediate System (IS-IS) protocol.
 *
 * Traffic: (please explain here what this traffic actually means, rather just 1 word)
 * L1 IS: 01:80:c2:00:00:14
 * L2 IS: 01:80:c2:00:00:15
 * All IS: 09:00:2b:00:00:05
 * 
 * Default packet action is forward.
```

Reply:
Done, Can you please review.

Reply:
> > > could the DCO be fixed?
> > 
> > 
> > Sure, Done.
> 
> Hi, it still shows DCO not yet passed? Thanks.



> > > could the DCO be fixed?
> > 
> > 
> > Sure, Done.
> 
> Hi, it still shows DCO not yet passed? Thanks.

Done. Please review.

Reply:
can you add spacing empty line between brief and default packet as proposed ? please also start each sentence with capital letters like Traffic word.

Reply:
Done. Thank you. Please review.